CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## 0.16.1
+
+BUG FIXES:
+* Invalidate JWT with single non-empty string aud on empty bound audiences https://github.com/hashicorp/vault-plugin-auth-jwt/pull/295
```

Comment:
Just a passing thought, we'll probably want to update the changelog on `main` as well? Since that's where most people will be looking for version and release information.

Reply:
Thank you. I will create a PR for `main` for the update.